docs-compose/overwatch_docs/content/DataEngineer/Productionizing.md

```diff
@@ -55,6 +56,224 @@ created each run but it will be replaced on each subsequent run. If you would li
+
+
+The snapshot data will be in the "targetPrefix/data" path as shown below:
+
+![Snapshot_Target_Data](/images/DataEngineer/snapshotTarget_Data.png)
+
+If you look into the targetPrefix folder you will find another subdirectory called "clone_report". This folder contains the data regarding the status of your snapshot process for each table:
+
+![clone_report](/images/DataEngineer/clone_report.png)
+
+##### Snapshot through Databricks Job
+
+You can also schedule a job to either run the notebook above, or use the SnapshotRunner class. For example:
+
+1. **Type** - Jar
+2. **Main class** - com.databricks.labs.overwatch.pipeline.SnapshotRunner
+3. **Parameters for the Job** - Same as mentioned above for [snapshot-through-databricks-notebook](#snapshot-through-databricks-notebook)
+
+![Snapshot_Job](/images/DataEngineer/snapshot_job.png)
+
+
+#### Validation Functionality
+This is a special module in Snapshot Process. 
+Before running your Snapshot, you can validate that your configuration is correct by calling the Snapshot.isValid function. This function is also used internally when the Snapshot is kicked off.
+
+Below is the code snippet for the validation function
+
+```scala
+import com.databricks.labs.overwatch.pipeline.Snapshot
+val sourceETLDB = "ow_bronze_snapshot_etl"
+val pipeline = "Bronze,Silver,Gold"
+val snapshotType = "Incremental"
+Snapshot.isValid(sourceETLDB,snapshotType,pipeline)
+```
+
+This function will return true if the configuration is valid, and will also output relevant info for the fields checked. For example:
+```
+ow_bronze_snapshot_etl is Overwatch Database and suitable for Snapshot.
+Snapshot Type is Suitable for Snapshot Process. Provided SnapshotType value is Incremental.
+cloneLevel Type is Suitable for Snapshot Process. Provided cloneLevel value is DEEP.
+Validation successful.You Can proceed with Snapshot process
+import com.databricks.labs.overwatch.pipeline.Snapshot
+res14: Boolean = true
+```
+
+### Restore Process
+You can use the Restore process to bring back snapshot data
+
+Similar to Snapshot, Restore can be run by both Databricks Job and Databricks Notebook.
+
+##### Restore through Databricks Notebook
+
+The code snippet to run the Restore process is as below:
+```scala
+import com.databricks.labs.overwatch.pipeline.Restore
+val sourcePrefix = "/mnt/overwatch_playground/721-snap/full"
+val targetPrefix = "/mnt/overwatch_playground/721-snap/restore"
+Restore.process(sourcePrefix,targetPrefix)
+```
+We need below parameters to run the Restore process through Databricks Notebook:
+
+| Param          | Type   | Optional  | Description                                                     |
+|----------------|--------|-----------|-----------------------------------------------------------------|
+| sourcePrefix   | String | No        | Source ETL Path Prefix from where restore need to be performed. |
+| targetPrefix   | String | No        | Target ETL Path Prefix where restore data will be loaded to.    |
+
```

Comment:
add an extra line

Reply:
Havn't changed anything here as both are looking same in WebUI Front